ONSdigital/dp-frontend-release-calendar
# Add data to the Release page before GTM Container Loaded event fires

### What

Use `DataLayerAdditions` to insert data into the GTM data layer before GTM loads and the Container Loaded event fires for tags.

### How to review

- Run the dp-design-system in a separate shell with `make debug`
- Run this frontend with `make debug`
- Visit any release page
- Examine GTM dashboard to verify data comes through on the Container Loaded event

### Who can review

Go / Frontend / Analytics developers
